Common/FeatureFlags.swift

```diff
@@ -90,13 +91,18 @@ struct FeatureFlagConfiguration: Decodable {
         #else
         self.includeServicesInSettingsEnabled = true
         #endif
-        
-        // Swift compiler config is inverse, since the default state is enabled.
+
         #if MOCK_THERAPY_SETTINGS_ENABLED
         self.mockTherapySettingsEnabled = true
         #else
         self.mockTherapySettingsEnabled = false
         #endif
+        
+        #if DELETE_ALL_BUTTON_DISABLED
```

Comment:
[nit] This is one that needs the comment deleted above. ;)

Reply:
Once you flip the logic as mentioned above.

Reply:
Last thing, I promise. Just add the "// Swift compiler config is inverse, since the default state is enabled." above this line. :)